python/setup.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
```

Comment:
Is this fallback really needed? Most Python modules just use setuptools nowadays.

Reply:
@mitya57 almost certainly not needed anymore, but most people who still use `distutils.core.setup` are ideologically opposed to removing it as a fallback

Reply:
I think distutils was here only for historical reasons.